executor/splittest/split_table_test.go

```diff
@@ -266,26 +267,37 @@ func TestShowTableRegion(t *testing.T) {
 	// 4 regions to store record data.
 	// 1 region to store index data.
 	require.Len(t, rows, 5)
-	require.Len(t, rows[0], 11)
+	require.Len(t, rows[0], 13)
 	tbl := external.GetTableByName(t, tk, "test", "t_regions")
 	// Check the region start key.
 	require.Equal(t, fmt.Sprintf("t_%d_r", tbl.Meta().ID), rows[0][1])
 	require.Equal(t, fmt.Sprintf("t_%d_r_-5000", tbl.Meta().ID), rows[1][1])
 	require.Equal(t, fmt.Sprintf("t_%d_r_0", tbl.Meta().ID), rows[2][1])
 	require.Equal(t, fmt.Sprintf("t_%d_r_5000", tbl.Meta().ID), rows[3][1])
 	require.Equal(t, fmt.Sprintf("t_%d_r", tbl.Meta().ID), rows[4][2])
+	// Check scheduling constraint and scheduling state default value
+	for i := range rows {
+		require.Equal(t, "", rows[i][11])
+		require.Equal(t, "", rows[i][12])
+	}
 
 	// Test show table index regions.
 	tk.MustQuery(`split table t_regions index idx between (-1000) and (1000) regions 4;`).Check(testkit.Rows("4 1"))
 	re = tk.MustQuery("show table t_regions index idx regions")
 	rows = re.Rows()
 	// The index `idx` of table t_regions should have 4 regions now.
 	require.Len(t, rows, 4)
+	require.Len(t, rows[0], 13)
 	// Check the region start key.
 	require.Regexp(t, fmt.Sprintf("t_%d.*", tbl.Meta().ID), rows[0][1])
 	require.Regexp(t, fmt.Sprintf("t_%d_i_1_.*", tbl.Meta().ID), rows[1][1])
 	require.Regexp(t, fmt.Sprintf("t_%d_i_1_.*", tbl.Meta().ID), rows[2][1])
 	require.Regexp(t, fmt.Sprintf("t_%d_i_1_.*", tbl.Meta().ID), rows[3][1])
+	// Check scheduling constraint and scheduling state default value
+	for i := range rows {
+		require.Equal(t, "", rows[i][11])
+		//require.Equal(t, new(infosync.PlacementScheduleState).String(), rows[i][12])
```

Comment:
Assert the empty schedule state, too

Reply:
seems like the code piece here is stale, it's in line 280-281, closing here, thanks!